src/components/common/TokenImg.tsx

```diff
@@ -35,7 +35,7 @@ const tokensIconsRequire =
   process.env.NODE_ENV === 'test' ? RequireContextMock : require.context('assets/img/tokens', false)
 
 const tokensIconsFilesByAddress: Record<string, string> = tokensIconsRequire.keys().reduce((acc, file) => {
-  const address = file.match(/0x\w{40}/)?.[0]
+  const address = file.match(/(0x\w{40}|eth|xdai)/)?.[0]
```

Comment:
i think learning from @Velenir 's input, `.test` is better/more efficient if this is simply a boolean check

Reply:
The thing is, we do need the extracted address. It's not used only for the conditional.

But I'll refactor the regex out just to make it a bit more efficient.